packages/dashboard-core-plugins/src/panels/MarkdownPanel.tsx

```diff
@@ -128,6 +130,10 @@ export class MarkdownPanel extends Component<
     }
   }
 
+  getClosedMarkdowns = memoize((closedPanels: ClosedPanels) =>
+    MarkdownUtils.getClosedMarkdowns(closedPanels)
+  );
```

Comment:
I think you could in theory just shorten this since it's a static function and you're just passing the args through, but no big deal:
```suggestion
  getClosedMarkdowns = memoize(MarkdownUtils.getClosedMarkdowns);
```

Reply:
Just changed it, I agree, looks a lot cleaner.